modules/config-updates-312.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: REFERENCE
+[id="config-updates-312"]
+= Configuration updates for {productname} 3.12
+
+The following sections detail new configuration fields added in {productname} 3.12.
+
+[id="registry-auto-prune-configuration-fields"]
+== Registry auto-pruning configuration fields
+
+The following configuration fields have been added to {productname} auto-pruning feature:
+
+.Registry auto-pruning configuration fields
+|===
+| Fields |Type |Description
+|*DEFAULT_NAMESPACE_AUTOPRUNE_POLICY* | Object | The default organization-wide auto-prune policy.
+|*.number_of_tags* | Integer | The number denoting the number of tags to keep. For example, to keep two tags, specify `2`.
```

Comment:
I think the examples below clear it up but technically under `DEFAULT_NAMESPACE_AUTOPRUNE_POLICY` there two supported key's `method` and `value` where `method = "number_of_tags" | "creation_date"` and `value = number | integer`. The description in this table makes it sound like the fields are `number_of_tags: 1` or `creation_date: 1y`. I would consider this a nit though.

Reply:
Same with the lines [here](https://github.com/quay/quay-docs/pull/1028/files#diff-963c4b8bbf0a6e6c02296cd8c25bcdac028460a666d894c825757e2fc70f60c5R33-R38).

Reply:
Fixed.